core/components/css-utilities/Position/Position.story.tsx

```diff
@@ -1,65 +1,445 @@
 import * as React from 'react';
 import { Heading, Card, Table, Text } from '@/index';
 import utilitiesSchema from '../Schema';
-import Paragraph from '@/components/atoms/paragraph';
 
 export const position = () => {
-  const data = [
+  const positionData = [
     {
       className: 'position-static',
-      properties: 'position: static ;',
+      properties: 'position: static !important;',
     },
     {
       className: 'position-relative',
-      properties: 'position: relative ;',
+      properties: 'position: relative !important;',
     },
     {
       className: 'position-absolute',
-      properties: 'position: absolute ;',
+      properties: 'position: absolute !important;',
     },
     {
       className: 'position-fixed',
-      properties: 'position: fixed ;',
+      properties: 'position: fixed !important;',
```

Comment:
Please add example for `position: fixed` as well 
<img width="1018" alt="image" src="https://github.com/user-attachments/assets/45751766-1490-48b9-839b-a4825bd8b4f2" />